ArduCopter/mode_guided.cpp

```diff
@@ -371,6 +371,13 @@ bool ModeGuided::set_destination(const Vector3f& destination, bool use_yaw, floa
     return true;
 }
 
+void ModeGuided::get_attitude_target(GCS_MAVLINK_Copter::Attitude_Target_Info &target) const
```

Comment:
Can you add a simple comment above the method?  Normally Copter has a comment above every method although I see that "get_wp" is missing this but that's an outlier.

Reply:
.. and sorry to be picky but let's try and keep the order of the definitions consistent with the order in the .cpp file.  So I think this should go below get_wp() (or the order should be changed in the .h